api/apps/api/src/modules/projects/dto/project-jobs-status.dto.ts

```diff
@@ -47,7 +47,7 @@ export class JSONAPIProjectJobStatusData {
   @ApiProperty({
     type: String,
   })
-  type = 'project-jobs';
+  type: 'project-jobs' = 'project-jobs';
```

Comment:
```suggestion
  type = 'project-jobs' as const;
```

Reply:
@Dyostiq this makes swagger go crazy and treat type as `Object`